snp_application/config/snp.btproj

```diff
@@ -24,6 +24,16 @@
             <input_port name="action_name" default="follow_joint_trajectory"/>
             <input_port name="trajectory" default="{trajectory}"/>
         </Action>
+        <Action ID="FreespaceMotionPlanPub" editable="true">
+            <input_port name="topic_name"/>
+            <input_port name="process" default="{process}"/>
+            <input_port name="trajectory" default="{trajectory}"/>
+        </Action>
+        <Action ID="GenerateFreespaceMotionPlanService" editable="true">
+            <input_port name="service_name" default="generate_freespace_motion_plan"/>
+            <input_port name="process" default="{process}"/>
```

Comment:
A generate FreespaceMotion service should specifically plan to/from joint states or some clearly defined 2 states. While I know the goal of this new task is to plan from the end of a process to the start of the process to enable repeat executions, this doesn't make the freespace planner flexible enough in my opinion.